### ISSUE TITLE
Request For Code

### ISSUE THREAD

**Issue by john-solon** [borisbanushev/stockpredictionai#38]
Could your please provide full code

**Comment by Earlyeon** [comment]
hello，i'm searching High Frequency Trading， can u please provide full code for giving me some help ,thank u very much,my email is wei895177311@qq.com

**Comment by pbezz1** [comment]
Hi all I'm seeing a lot of requests for code, but are you actually receiving a response from the author?

Thanks

**Comment by Earlyeon** [comment]
No, I ask for requests for code this morning ,but i still didn't receive  a response util now.




------------------ Original ------------------
